projectfiles/Xcode/The Battle for Wesnoth.xcodeproj/project.pbxproj

```diff
@@ -3915,7 +3924,7 @@
 				B2CC45FEA71445AE817CAA6B /* edit_pbl.hpp */,
 				C679447D91FD3623CC852FF8 /* edit_pbl_translation.hpp */,
 				C61F473D9AC43768A445E218 /* tod_new_schedule.cpp */,
-				5D46466DBCD81B13621C7342 /* tod_new_schedule.hpp */,
+				5D46466DBCD81B13621C7342 /* tod_new_schedule.hpp */
```

Comment:
I don't have Visual Studio to test with, but the trailing comma on the last line of the other blocks makes me think it should be here too.

Reply:
This is for xcode though?

Reply:
Adjusted.